flixel/system/debug/console/Console.hx

```diff
@@ -17,7 +17,6 @@ import openfl.events.FocusEvent;
 import openfl.events.KeyboardEvent;
 import openfl.text.TextFieldType;
 import openfl.ui.Keyboard;
-
```

Comment:
could you add `"editor.formatOnSaveMode": "modifications"`  to your settings.json and `"trailingWhitespace": true` to your hxformat.json, for future flixel PRs?

Reply:
note, I had to put `"trailingWhitespace": false` or else it was inputting tabs as whitespace after each line with whitespace! but all whitespacing related accidental changes should be fixed